docs/en/operation/Journal.md

```diff
@@ -184,9 +184,10 @@ This functionality is enabled by setting the following property in `${ALLUXIO_HO
 alluxio.master.daily.backup.enabled=true
 ```
 
-The time to take daily snapshots is defined by `alluxio.master.daily.backup.time`. For example, if
-a user specified `alluxio.master.daily.backup.time=05:30`, the Alluxio leading master will back up its metadata
-to the `alluxio.master.backup.directory` of the root UFS every day at 5:30am UTC.
+The time to take daily snapshots is defined by `alluxio.master.daily.backup.time` and `alluxio.master.daily.backup.interval`. 
+For example, if a user specified `alluxio.master.daily.backup.time=05:30` and `alluxio.master.daily.backup.interval=6hr`, 
+the Alluxio leading master will back up its metadata to the `alluxio.master.backup.directory` of the root UFS, 
+the first day is 5:30am UTC, followed by an interval of 6 hours.
```

Comment:
This is a bit strange. By `daily.backup.xxx` it infers this backup operation happens daily (by 24hrs interval)? So having the interval be 6hrs means it's no longer "daily". Also if the interval is `5hrs` then what does it do? It's going to be the 1st `5:30am` then `10:30am` then `3:30pm` then `8:30pm` then `1:30am` then `6:30am`, you see the `5:30am` is gone and it's quite confusing.

Do you need backup a few times a day? That is a really big operation.

If you really do, i figure we can make the properties `alluxio.master.backup.start.time` and `alluxio.master.backup.interval`. `alluxio.master.backup.start.time` either defaults to or alias to `alluxio.master.daily.backup.time=05:30`.

WDYT @jenoudet ?